copy-json-to-bipf.js

```diff
@@ -1,6 +1,6 @@
 var pull = require('pull-stream')
 var FlumeLog = require('flumelog-offset')
-var FlumeLogRaf = require('../')
+var FlumeLogAligned = require('flumelog-aligned-offset')
```

Comment:
`require` of `../` didn't make sense to my local folder structure when cloning this repo

Reply:
right, this was because it was copied over from flumelog-aligned-offset repo and never properly fixed.